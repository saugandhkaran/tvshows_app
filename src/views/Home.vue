<template>
  <div class="home">
    <div class="flex-container center">
      <p>Welcome to PLAYTV24x7</p>
    </div>
    <div class="flex-container center">
      <router-link to="/search"><button class="btn-secondary">Search</button></router-link>
      <router-link to="/browse"><button class="btn-primary">Browse</button></router-link>
    </div>
  </div>
</template>

<script>
export default {
  name: 'Home'
}
</script>

<style lang="scss" scoped>
p {
  font-size: 4em;
  font-weight: bold;
  text-align: center;
}
</style>
