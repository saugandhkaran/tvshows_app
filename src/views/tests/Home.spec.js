import { shallowMount, RouterLinkStub } from '@vue/test-utils';
import Home from '../Home';

describe('Home', () => {
  
  it('should match the Home html', () => {
    const wrapper = shallowMount(Home, {
      stubs: {
        RouterLink: RouterLinkStub,
      },
    })
    expect(wrapper.element).toMatchSnapshot();
  });
  
});
