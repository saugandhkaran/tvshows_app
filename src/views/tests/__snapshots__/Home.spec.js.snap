// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home should match the Home html 1`] = `
<div
  class="home"
>
  <div
    class="flex-container center"
  >
    <p>
      Welcome to PLAYTV24x7
    </p>
  </div>
   
  <div
    class="flex-container center"
  >
    <a>
      <button
        class="btn-secondary"
      >
        Search
      </button>
    </a>
     
    <a>
      <button
        class="btn-primary"
      >
        Browse
      </button>
    </a>
  </div>
</div>
`;
